src/components/Forecast.tsx

```diff
@@ -10,42 +10,48 @@ interface Props {
 	units?: Unit
 }
 
+function useTemperatureConverter() {
+	return function(temp: number, toUnit:Unit):number{
+		switch (toUnit) {
+		      case Unit.Imperial: 
+		      	 temp=Math.floor((temp * 9/5)+32);
+				 break;
+		      default: 
+		      	 temp=Math.floor(temp + 273.15);
+				 break;
+		}
+		return temp;
+	}
+  
+}
+
 export const Forecast: React.FC<Props> = ({forecast, units}) => {
 	
-	const [newUnits,setNewUnits]=useState<string>('ºC');
+	const [tempUnits,setTempUnits]=useState<string>("ºC");
+	
+	const [temp,setTemp]=useState<number>(Math.floor((Math.floor(forecast.main.temp_max)/Math.floor(forecast.main.temp_min))));
```

Comment:
I think you misunderstood me. The app has to show both min and max temperatures, NOT divide them. Dividing it makes no sense.